pandas/core/strings.py

```diff
@@ -1095,24 +1095,48 @@ def str_pad(arr, width, side='left', fillchar=' '):
 
 def str_split(arr, pat=None, n=None):
     """
+    Split strings around given separator/delimiter.
+
     Split each string (a la re.split) in the Series/Index by given
```

Comment:
Can put reference to `re` module in backticks, so \`re.split\`

Reply:
It's already in the extended summary.
```
Split each string (a la re.split) in the Series/Index by given pattern, propagating NA values. Equivalent to :meth:`str.split`.
```

Reply:
Right I just mean actually surrounding the reference with backticks so it renders as inline code. So simply change any occurrence of re.split to \`re.split\`